odbc-api/src/handles/connection.rs

```diff
@@ -186,4 +186,41 @@ impl<'c> Connection<'c> {
                 .into_result(self)
         }
     }
+
+    /// Fetch the name of the database management system used by the connection and store it into
+    /// the provided `buf`.
+    pub fn fetch_database_management_system_name(&self, buf: &mut Vec<u16>) -> Result<(), Error> {
+        // String length in bytes, not characters. Terminating zero is excluded.
```

Comment:
Let me know if we should try to share some of this implementation with https://github.com/pacman82/odbc-api/blob/f1a3cd29357dd4bd105cd051655e14d64bfb1758/odbc-api/src/handles/statement.rs#L737 somehow

Reply:
`col_name` works on statement handles rather than connections, so I guess any means of sharing would introduce an additional indirection, which arguably would make it harder to read.